src/store/modules/models/libconfig.js

```diff
@@ -379,7 +379,7 @@ const map = {
         displayName: 'Construction Set',
         input_type: 'select',
         select_data(space, state) {
```

Comment:
I wonder if there's a good way to abstract out some of this duplicate code?

Reply:
But go ahead and merge if you don't think it's something worth messing with right now :)